srcCpp/Infrastructure_pro.cxx

```diff
@@ -538,9 +570,9 @@ bool configureShmemTransport(
     RTIBool success = RTI_FALSE;
     int retcode;
     int key = rand();
-    int minBufferSize = parentMsgSizeMax;
-    int step = 1048576 + parentMsgSizeMax; // 1MB + parentMsgSizeMax
-    int maxBufferSize = 60817408; // 58MB
+    int minBufferSize = 1048576;
```

Comment:
This looks wrong to me. This should be the minimum size of a message, not 1MB

Reply:
@fercsrti77, I am changing this myself prior to the merge, lets review it later.